modules/deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode.adoc

```diff
@@ -0,0 +1,256 @@
+:_module-type: PROCEDURE
+
+[id="deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_{context}"]
+= Deploying models on single node openshift using kserve raw deployment mode
+
+[role='_abstract']
+You can deploy a machine learning model by using KServe raw deployment mode on single node OpenShift. With raw deployment mode, you can overcome the limitations of Knative, such as gaining the ability to mount multiple volumes.
```

Comment:
Is the ability to mount multiple volumes a limitation?

Reply:
Changed to your suggestion, thanks!